config.json.example

```diff
@@ -17,7 +17,11 @@
             500,
             1000,
             2000
-        ]
+        ],
+        "channels": {
+            "guild1": "channel1",
+            "guild2": ["channel2", "channel3"]
+        }
```

Comment:
I'm debating whether or not to open this up to other commands but for now, I'm just implementing it for these because people are using them in other channels

Reply:
Is it possible to make this config key any clearer? Having it just be "channels" is confusing to me

Reply:
`allowedChannels`?

Reply:
Maybe something like 'commandChannels'